bug-bounty-list/bug-bounty-list.json

```diff
@@ -7608,5 +7608,25 @@
     "swag": false,
     "hall_of_fame": true,
     "safe_harbor": ""
+  },
+   {
+    "program_name": "Edmodo",
+    "policy_url": "",
+    "submission_url": "privacy@edmodo.com",
+    "launch_date": "",
+    "cash_reward": false,
+    "swag": true,
```

Comment:
Where do you see any page/terms that outline that swag is rewarded for findings? I saw https://go.edmodo.com/privacy-policy/ but couldn't find anything concrete.

Reply:
It is not mentioned but I have received twice and many other hackers have received it many time :)